sw/device/silicon_creator/lib/cert/cbor_helper.h

```diff
@@ -0,0 +1,75 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CERT_CBOR_HELPER_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CERT_CBOR_HELPER_H_
+
+#include "include/dice/cbor_writer.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CBOR_CHECK_OVERFLOWED_AND_RETURN(p) \
```

Comment:
undef this at the end of this file? https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros